handbook.md

```diff
@@ -236,6 +236,61 @@ As a maintainer, your responsibility is to keep a project running, taking care o
 
+## Becoming a committer
+#### What is a committer
+
+A Foreman committer is a person with commit access to one or more of the repositories under [the Foreman organization](https://github.com/theforeman/). Committers are members of the Foreman contributors community who exhibit most of the following behaviors:
+
+* Review and merge code and documentation
+* Help triaging bugs and testing pull requests
+* Make well formed pull requests
+* Take care of releases
+* Have a sense of duty about the Foreman project
+* Play well with others, are respectful, show gratitude
+
+We ask new Foreman contributors to act as a committer to the extent capable. It's the best route to become a committer if that is your goal, but even if it's not, it makes our community better.
+
+If you become a committer, we expect of you to:
+
+*  Help out users and other developers on the mailing lists, in Redmine, Serverfault, and in IRC #theforeman and #theforeman-dev
+*  Review and test the patches submitted by others; this can help to offload the burden on existing committers, who will definitely appreciate your efforts
+*  Participate in discussions about releases, roadmaps, architecture, and long-term plans
+*  Help improve the website
+*  If possible, participate in (or even initiate) real-world events such as user/developer meetups, papers/talks at conferences, etc
+*  Improve project infrastructure in order to increase the efficiency of committers and other contributors
+*  Help raise the project's quality bar (e.g. by setting up code coverage analysis)
+*  As much as possible, keep your activity sustained rather than sporadic
+
+#### How do I become a committer?
+
+One person has to nominate you to the group of existing committers.
+The person who nominates you has to:
+
+  * Submit 10 examples that prove this person behaves like a committer. For instance:
+    * Strategic patches to the project that fix or add key functionality
+    * Mailing list threads where the person is clearly helpful
+    * Comments that proof the person has triaged bugs
+    * Proof the nominee has maintained a plugin successfully
+    * Anything else that shows the nominee should become a committer
+  * Explain how the nominee is invoved in the community and cares about the future of the project
+
+After the nomination is submitted, two other committers have to second your nomination. If no one objects in one week, you're a committer.
+If anyone objects or wants more information, that person needs to argue for the objection. After that, the committers discuss and come to a consensus within the next week.
+
+In the event of a failed nomination, feedback is reported back to the nominee to help them improve. Feedback comes from a spokesperson from the whole committers group, not just an individual, to avoid potential rancor between the rejected nominee and an existing committer.
```

Comment:
I think this is really two separate things. There's no reason for the nomination not to be on -dev (that seems to have been fine recently for @tbrisker and @orrabin). The candidates will in some respect become a part of the project, and should be comfortable discussing things properly on -dev, especially positive things like this ;)

People rarely wish to discuss negative things publicly though, I agree. However, an extra list won't help, I don't think. Should there be an issue with a nomination that cannot be discussed on the thread, I would propose this process:

* Member with concerns speaks privately to the sponsor to discuss the issue.
* If the issue remains, sponsor posts to the thread something like "Private concerns have been raised with this nomination, so I am withdrawing it until they can be resolved"
* Sponsor can then work with candidate to resolve the issue(s).

This removes the additional bureaucracy of an an extra list, and allows constructive feedback to come from someone who already feels positive about the candidate. It does require trust that these private conversations are done in a correct manner, but that's true of the many ways we already communicate with each other already.